src/components/containers/SingleSidebarItem.js

```diff
@@ -1,18 +1,10 @@
 import PropTypes from 'prop-types';
-import React, {cloneElement, Component} from 'react';
-import Button from '../widgets/Button';
+import React, {Component} from 'react';
 
 export default class SingleSidebarItem extends Component {
   render() {
-    let {children} = this.props;
-    children = React.Children.map(children, child => {
-      if (child.type === Button) {
-        return cloneElement(child, {className: 'button--menu'});
```

Comment:
but wait, aren't we losing some special styles on the buttons?

Reply:
Yes, but as I said in the header, this is the wrong place to do it... if you want special styles on your button, you should do it either in a CSS selector, or in the `<Button className={}>` but not magically here...

Reply:
We can't start adding special cases for every widget type that someone wants to jam into a sidebar :)

Reply:
We could fix this by using css rather than js to denote if its contained within the sidemenu, eg

```
.button{
  @at-root .sidemenu &{
    ... changes
  }
}
```

Reply:
oh oups, was not seeing these..ok for the css

Reply:
@aulneau I'm actually not familiar with this `@at-root` directive... would it be possible for you to tack on a commit to this PR that does this please? :)

Reply:
sure thing!

Reply:
🙇  normally I'm not one to skip on learning new stuff but I've got a few other things going on all of a sudden that people want my input on :)

Reply:
Okay! should be good to go @nicolaskruchten